README.md

```diff
@@ -1,2 +1,58 @@
-# jupyterhub-ssh
-SSH Access to JupyterHubs
+# JupyterHubSSH
+
+[![Documentation build status](https://img.shields.io/readthedocs/jupyterhub?logo=read-the-docs)](https://jupyterhub-ssh.readthedocs.io/en/latest/)
+
+SSH Access to JupyterHubs, independent of the way there were deployed.
+
+## Development Status
+This project is under active develpoment :tada:, so expect a few changes along the way.
+
+## Technical Overview
+
+JupyterHubSSH is made up of two main components:
+
+- a Notebook SSH server that maps a SSH connection to a Notebook server on a JupyterHub.
+- a [Terminado](https://github.com/jupyter/terminado) client that knows how to connect and communicate to a Jupyter terminal.
+
+![Overview](https://raw.githubusercontent.com/yuvipanda/jupyterhub-ssh/main/docs/source/_static/images/technical-overview.png)
+
+## Installation
+
+1. Clone the repo and install the jupyterhub-ssh package:
+``` bash
+$ git clone https://github.com/yuvipanda/jupyterhub-ssh.git
+$ cd jupyterhub-ssh
+$ pip install .
```

Comment:
```suggestion
$ pip install -e .
```

Do we want to suggest an editable install here, so changes to the source will be reflected after install?

Reply:
Why not? :+1: